deploy/kubernetes/overlays/noauth/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
As mentioned above I think we should combine the noauth into dev and just have dev.

Reply:
Done